### DISCUSSION TITLE
Offer total metric (instead of/additional to components)

### DISCUSSION THREAD

**Discussion by jochen-schuettler** [sustainable-computing-io/kepler#1242] [Ideas]
Kepler offers metrics in form of components (PKG,DRAM,OTHER). While it's nice to have that distinction, the total curve, summing up all three would also be interesting. Ok, the standard Kepler exporter dashboard shows all three in a cumulative view, so the upper curve (OTHER) IS the total. But that only gives the total sum of all. Pod-wise (or service-wise, see my other proposal) totalizition improves the visual comparison between the pods (or services).
I can build such sums in Grafana, but with the graphic editor it's time-consuming and error-prone. I would like a programmatic solution that gets exported and visualized in the standard Kepler dashboard.

**Comment by rootfs** [comment]
You can use `kepler_container_joules_total`, it sums all (pkg, dram, uncore and gpu, if used).

**Comment by jochen-schuettler** [comment]
works - Thank you!
